modules/gdscript/language_server/gdscript_language_protocol.cpp

```diff
@@ -43,6 +43,8 @@ void GDScriptLanguageProtocol::on_data_received(int p_id) {
 	if (OK == peer->get_packet_buffer(data)) {
 		String message;
 		message.parse_utf8((const char *)data.read().ptr(), data.size());
+		std::size_t json_start = message.find("{");
```

Comment:
The build fails https://ci.appveyor.com/project/akien-mga/godot/builds/29805069

The type is also wrong, `String::find` returns an `int`, not a `size_t`.

Reply:
We'd better follow the LSP specification.

> The header part consists of header fields. Each header field is comprised of a name and a value, separated by ‘: ‘ (a colon and a space). Each header field is terminated by ‘\r\n’. Considering the last header field and the overall header itself are each terminated with ‘\r\n’, and that at least one header is mandatory, this means that two ‘\r\n’ sequences always immediately precede the content part of a message.

You can get the content body start position by `message.find("\r\n\r\n")`